openquake/engine/engine.py

```diff
@@ -241,12 +241,14 @@ def run_calc(log):
     setproctitle('oq-job-%d' % log.calc_id)
     with log:
         # check the available memory before starting
+        """
         while True:
             used_mem = psutil.virtual_memory().percent
             if used_mem < 50:  # continue if little memory is in use
                 break
             logging.info('Used memory %d%%, waiting', used_mem)
             time.sleep(5)
+        """
```

Comment:
I have solved the issue in master; just update the branch.

Reply:
Will do ... for the time being my headaches are elsewhere .. :)